locales/en.default.schema.json

```diff
@@ -1063,6 +1063,15 @@
         "show_line_separator": {
           "label": "Show separator line"
         },
+        "header__1": {
+          "content": "Color"
+        },
+        "header_color_scheme": {
+          "label": "Header color scheme"
```

Comment:
@YoannJailin I don't feel strongly at all, but just wondering if we don't want to keep the original "Color scheme" label for the header's color scheme setting. I could certainly see this adding some clarity for users, but the setting is within the "Header" section as it is, so I somewhat feel like we don't need the additional qualifier.

We only have a couple other instances of multiple color schemes in one section to pull precedent from but they just use "Color scheme" to represent the section's color scheme.

<img src="https://screenshot.click/11-56-drdy9-qfkv4.png" width="300" />
<img src="https://screenshot.click/11-57-wqsr2-a92yv.png" width="300" />

Reply:
Yes i agree we can replace "Header color scheme" by "Color scheme". I would keep the second one named "Menu color scheme". @lougoncharenko @kmeleta

Reply:
Changed it back to the original translation and removed the header_color_scheme translation